zanata-war/src/main/java/org/zanata/service/impl/LocaleServiceImpl.java

```diff
@@ -89,11 +89,15 @@ public LocaleServiceImpl(LocaleDAO localeDAO, ProjectDAO projectDAO,
     }
 
     public static Map<LocaleId, String> getLocaleAliasesByIteration(HProjectIteration iteration) {
-        Map<LocaleId, String> localeAliases;
-        if (iteration.isOverrideLocales()) {
-            localeAliases = iteration.getLocaleAliases();
-        } else {
-            localeAliases = iteration.getProject().getLocaleAliases();
+        Map<LocaleId, String> localeAliases = null;
```

Comment:
Don't think this will solve the nullpointer exception. I think we can return `localeAliases` in the try statement. Then return `new HashMap<LocaleId, String>` at the end of the method to prevent the exception.